Controllers/AdminController.cs

```diff
@@ -30,16 +30,26 @@ public ActionResult Search(SearchVM searchVM)
                 //Filter the collection by Title which "contains" the given string
                 foundBooks = foundBooks
                              .Where(b => b.Title.Contains(searchVM.Title));
+                               .OrderBy(b => b.Title(searchVM )); //Coundn't manage to get this stuff to work
```

Comment:
The `OrderBy()` method uses its lambda expression to identify which  property to use as it sorts the collection, this looks like `OrderBy(b => b.Title);`

If you get an error saying that you need to do this "on the client-side" then move the `OrderBy()` method to the  View, `SearchResults.cshtml` to the part of the code where you define which collection your `foreach` loop is reading from. That collection can be sorted using `OrderBy()`.